jpegenc/models/rgb2ycbcr/rgb2ycbcr.py

```diff
@@ -0,0 +1,156 @@
+#!/bin/python
+from myhdl import block, Signal, ResetSignal, intbv, delay, instance, always_comb, always_seq
+from myhdl.conversion import analyze
+
+Y_COEFF=[0.2999,0.587,0.114]
+CB_COEFF=[-0.1687,-0.3313,0.5]
+CR_COEFF=[0.5,-0.4187,-0.0813]
+OFFSET=[0,128,128]
+
+def build_coeffs(fract_bits):
+
+    Y=[int(round(Y_COEFF[i]*(2**fract_bits )))for i in range(3)]
+    Cb=[int(round(CB_COEFF[i]*(2**fract_bits ))) for i in range(3)]
+    Cr=[int(round(CR_COEFF[i]*(2**fract_bits ))) for i in range(3)]
+    Offset=[int(round(OFFSET[i]*(2**fract_bits ))) for i in range(3)]
+
+    return Y,Cb,Cr,Offset
+
+
+class RGB(object):
+
+    def __init__(self, nbits=8):
+        self.nbits=nbits
+        self.red = Signal(intbv(0)[nbits:])
+        self.green = Signal(intbv(0)[nbits:])
+        self.blue = Signal(intbv(0)[nbits:])
+
+    def bitLength(self): return self.nbits
+
+
+class YCbCr(object):
+
+    def __init__(self, nbits=8):
+        self.nbits = nbits
+        self.y = Signal(intbv(0)[nbits:])
+        self.cb = Signal(intbv(0)[nbits:])
+        self.cr = Signal(intbv(0)[nbits:])
+
+    def bitLength(self): return self.nbits
+
+@block
+def rgb2ycbcr(ycbcr, enable_out, rgb, enable_in, clk, reset, fract_bits = 14, nbits = 8):
```

Comment:
We need to add doc strings to each ~~module~~ block use the [google style](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html) doc strings for the blocks and interfaces.

It should be very clear what this block does, what the inputs and outputs are, etc.